clients/ui/bff/internal/integrations/mrserver/http.go

```diff
@@ -218,6 +226,16 @@ func (c *HTTPClient) PATCH(url string, body io.Reader) ([]byte, error) {
 	return responseBody, nil
 }
 
+func (c *HTTPClient) applyHeaders(req *http.Request) {
+	if c.Headers != nil {
```

Comment:
@lucferbux this can be done in a FUP PR, but I would not fw all headers here, but just the ones that are necessary (for security reasons). Can you add this as a TODO for the future?